src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -855,17 +855,19 @@ def getTrustSitelists(self):
         """
         _getTrustSitelists_
 
-        Accessor for the 'trust site lists' flag for the task.
+        Get the input and pileup flag for 'trust site lists' in the task.
         """
-        return self.data.constraints.sites.trustlists
+        return (self.data.constraints.sites.trustlists,
+                self.data.constraints.sites.trustPUlists)
```

Comment:
I wonder it might be clearer to return dict instead of tuple.  (changes others according to that).

```
{"trustlists":self.data.constraints.sites.trustlists,
 "trustPUlists": self.data.constraints.sites.trustPUlists}
```